ui/src/tool-panel/location-track/location-track-infobox.tsx

```diff
@@ -59,6 +60,8 @@ type LocationTrackInfoboxProps = {
     viewport: MapViewport;
     visibilities: LocationTrackInfoboxVisibilities;
     onVisibilityChange: (visibilities: LocationTrackInfoboxVisibilities) => void;
+    onVerticalGeometryDiagramVisibilityChange: (visibility: boolean) => void;
```

Comment:
Näissä ylätason komponenteissa kasvaa helposti nämä parametrimäärät aika isoiksi. En ole varma kuinka paljon sitä kannattaa harrastaa, mutta periaatteessa yhteen alakomponenttiin kuuluvilla propseilla (kuten tämä verticalgeometry) tuota voi kiertää tekemällä alakomponentille oma container ja bindata redux-store suoraan siellä jolloin noita parametreja ei tarvitse välittää parentin läpi ja komponentti on itsenäisempi/uudelleenkäytettävämpi. 

Mielipiteitä/linjauksia @ksurakka-solita ?

Reply:
Containereita kandee hyödyntää niissä kohtaa, missä  jokin propsu vaan välitetään etiäpäin. Monet propsuthan vaan valuu suoraan yläkomponentilta alakomponenteille